tutor/src/components/task/progress/panel.cjsx

```diff
@@ -71,16 +71,17 @@ ProgressPanel = React.createClass
     titles = TaskPanelStore.getTitlesForStepIndex(@props.taskId, this.props.stepKey)
     className = classnames('progress-panel', 'page-loading': isCompleting or isLoading)
 
-    <PagingNavigation
-      className={className}
-      enableKeys={@props.enableKeys}
-      isForwardEnabled={@state.shouldShowRight}
-      isBackwardEnabled={@state.shouldShowLeft}
-      onForwardNavigation={@goForward}
-      onBackwardNavigation={@goBackward}
-      titles={titles}
-    >
-      {@props.children}
-    </PagingNavigation>
-
+    <ObscuredPage>
```

Comment:
would it be better to put this in `components/app.jsx` so that it can be used on any screen without having to include it?

Reply:
hm that's an interesting idea.  I could see us needing that in the future, but right now the overlay should only render if there's a page that supports overlaying (reading steps & ref view).

We'd need to generalize it more so that the overlay would know more about the current page if we move to app.